containers/proxy-helm/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 # SPDX-License-Identifier: MIT
 #!BuildTag: uyuni/proxy:latest
-#!BuildTag: uyuni/proxy:%PKG_VERSION%
-#!BuildTag: uyuni/proxy:%PKG_VERSION%-build%RELEASE%
+#!BuildTag: uyuni/proxy:4.3.1-1
```

Comment:
You should probably align with the current version of the SUSE Manager images which is `4.3.3` as of now.